docs/source/user_guide/tutorials/getting_started.rst

```diff
@@ -115,623 +392,114 @@ Let's use the alias to print the results parquet, the location of which was prin
+input record it was able to link to another record for our probability threshold 
+of ``.997``. ``Cluster ID`` names are chosen by Splink based on the first record 
+assigned to them.
+
+Running the pipeline also generates a :download:`DAG.svg <DAG-naive-pipeline.svg>` file in 
+the results directory which shows the implementations, data dependencies and 
+input validations present in the pipeline.
+
+To see how the model linked records before assigning them to clusters, we can 
+look at the intermediate output produced by the ``splink_evaluating_pairs`` 
+implementation::
+
+    $ pqprint results/2025_06_17_10_40_24/intermediate/splink_evaluating_pairs/result.parquet
+            Left Record Dataset  Left Record ID Right Record Dataset  Right Record ID  Probability
+    0       input_file_ssa_2020           16314   input_file_w2_2020             7604      0.00057
+    1       input_file_ssa_2020           16318   input_file_w2_2020             7604      0.00057
+    2       input_file_ssa_2020           16326   input_file_w2_2020             6049      0.00057
+    3       input_file_ssa_2020           16351   input_file_w2_2020             3549      0.00057
+    4       input_file_ssa_2020           16353   input_file_w2_2020             7434      0.00057
+    ...                     ...             ...                  ...              ...          ...
+    515790  input_file_ssa_2020           16309   input_file_w2_2020             7604      0.00057
+    515791  input_file_ssa_2020           16310   input_file_w2_2020             7604      0.00057
+    515792  input_file_ssa_2020           16311   input_file_w2_2020             7604      0.00057
+    515793  input_file_ssa_2020           16312   input_file_w2_2020             7604      0.00057
+    515794  input_file_ssa_2020           16313   input_file_w2_2020             7604      0.00057
+
+    [515795 rows x 5 columns] 
+
+The record pairs displayed in the preview are all far below the match threshold, but the results could 
+be investigated further using ``pandas.read_parquet()`` in a Python session.
+
+The Splink implementations in our pipeline also produce some diagnostic charts which can be useful 
+for evaluating results, such as the :download:`match weights chart<naive_match_weights.html>` and 
+:download:`comparison viewer tool<naive_comparison_viewer.html>`. These charts are available in the 
+``diagnostics/splink_evaluating_pairs`` subdirectory of the results directory for each pipeline run.
+
+Finally, since we generated the input datasets ourselves, and therefore know the ground truth of 
+which records are truly links, lets see how our naive model performed. For a threshold match
+probability of ``.997`` (chosen using match rate evaluation metrics), out of ``9262`` true links, 
+we can calculate that our model results contained ``260`` false positives and ``43`` false negatives, 
+with the rest of the links being accurate.
```

Comment:
```suggestion
probability of ``.997``, out of 9,262 true links to be found our model missed 43 (false negatives),
and additionally linked 260 pairs that shouldn't have been linked (false positives).
Depending on our goals with the linked data, we might increase the threshold to reduce false positives,
at the cost of increased false negatives.
But this was a simple linkage model.
Let's improve it to see if we can get a better performance tradeoff!
```

I don't think you need to specify how the threshold was chosen. My suggestion also makes the false negative/positive definitions a bit more explicit